### DISCUSSION TITLE
Guess country

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#681] [Galaxy Trivia] [answered]
🟦🟨🟥
🟨⬜⬛🤲
🟧⬜🟠🟩

**Comment by SANKARAMDAS** [comment, marked as answer]
Romania
Brunei
Niger

**Comment by peterbaccaro** [comment]
1. ROMANIA
2. BRUNEI
3. NIGER

![flag_2](https://user-images.githubusercontent.com/74666079/236122205-779f71f9-99ad-4cfb-b106-250d77de687d.png)
